index.html

```diff
@@ -14,120 +14,75 @@
 
 	<section class="container">
 
-	<!-- Menu Content-->
-	<header>
-		<h1>S&J</h1>
-		<button><img src="img/nav-hamburger.png" alt="navbarImg"></button>
-		<div class="drop-menu">
-			<a href="#">Home</a>
-			<a href="#">Services</a>
-			<a href="#">Contact</a>
-		</div>
-	</header>
-
-	<!-- Home Page Content-->
-	<div class="main-top">
-		<img src="img/home/home-jumbotron.png">
-		<p>Integrity, <br> Excellence, <br> Progress.</p>
-	</div>
+		<!-- Menu Content-->
+		<header>
+			<h1>S&J</h1>
+			<button><img src="img/nav-hamburger.png" alt="navbarImg"></button>
+			<div class="drop-menu">
+				<a href="#">Home</a>
+				<a href="#">Services</a>
+				<a href="#">Contact</a>
+			</div>
+		</header>
```

Comment:
Consider adding the routes to the `href attribute` as you construct the anchor tags. It will safe you `time.
			```<a href="index.html">Home</a>
			<a href="services.html">Services</a>
			<a href="contact.html">Contact</a>```